styleguide/derek/_partials/_ceiling.ejs

```diff
@@ -5,13 +5,13 @@
           <nav class="ceiling-menuNav">
               <ul class="ceiling-menuInline" role="menu">
                 <li class="ceiling-menuInline-item">
-                  <a class="navbar-phoneicon" title="Sales/Support Call"  data-toggle="modal" data-target="#myModal">Call Us</a>
+                  <a class="navbar-phoneicon" title="Sales/Support Call"  data-toggle="modal" data-target="#myModal"><span class="hidden-xs">Call Us</span></a>
```

Comment:
We're already using `hidden-xs` further down the file, so I'm not too worried about it here. If we ever refactor this in the future, then maybe we can deal with that then for now, I'm okay with this.